### PR TITLE
tools: Add TiDB Controller usage

### PR DESCRIPTION
**Pull request by jackysp**
Add TiDB Controller usage. PTAL @lilin90

### CODE REVIEW

File: tools/tidb-controller.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+* `tidb-ctl mvcc` - MVCC 信息
+* `tidb-ctl region` - Region 信息
+* `tidb-ctl schema` - Schema 信息
+* `tidb-ctl table` - Table 信息
+
+### 使用举例
+
+以获取 Schema 信息为例：
+
+通过 `tidb-ctl schema -h` 可以获取这个子命令的使用帮助。schema 有两个子命令，in 和 tid 。 in 用来通过数据库名获取数据库中所有表的表结构， tid 用来通过全数据库唯一的 table_id 获取表的表结构。
```
Comment (lilin90) [inline tools/tidb-controller.md]: Ditto. Delete the following extra whitespaces:
- After the 1st "tid"
- Before the 2nd "in"
- Before the 2nd "tid"

File: tools/tidb-controller.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+#### in 命令
+
+同样可以通过 `tidb-ctl schema in -h/--help` 来获取子命令 in 的使用帮助。
+
+##### 基本用法
+
+`tidb-ctl schema in {数据库名}`
+
+如： `tidb-ctl schema in mysql` 将得到以下结果：
```
Comment (lilin90) [inline tools/tidb-controller.md]: Ditto. Delete the extra whitespace after "如：".

File: tools/tidb-controller.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        "Partition": null
+    }
+]
+```
+
+结果将以 json 形式展示，内容较长，这里做了截断。
+
+如希望指定表名，可以使用 `tidb-ctl schema in {数据库名} -n {表名}` 进行过滤。
+
+如： `tidb-ctl schema in mysql -n db` 将得到 mysql 库中 db 表的表结构，结果如下：
```
Comment (lilin90) [inline tools/tidb-controller.md]: Ditto. Delete the extra whitespace after "如：".

File: tools/tidb-controller.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        "L": "db"
+    },
+    ...
+    "Partition": null
+}
+```
+
+这里同样做了截断。
+
+如希望指定服务地址，可以使用 `-H -P` 选项，如： `tidb-ctl -H 127.0.0.1 -P 10080 schema in mysql -n db` 。
```
Comment (lilin90) [inline tools/tidb-controller.md]: Ditto. Delete the two extra whitespaces:

- After "如：" 
- Before the period

**Comment by jackysp** [comment]
There is a strange bug in my VSCode. PTAL again, @lilin90

**Comment by jackysp** [comment]
PTAL @lilin90

**Comment by jackysp** [comment]
PTAL @shenli

**Pull request merged**